app/controllers/manage/questions_controller.rb

```diff
@@ -12,16 +12,23 @@ def create
   def filter
     @questions = Question.search_question(params)
     @questions = @questions.paginate(:page => params[:page] || 1, :per_page => 15)
-    render :index
+    if params[:from] == "question_crud"
+      render '/manage/admins/question_crud'
+    else
+      render :index
+    end
   end
```

Comment:
@Iulia-Soimaru so i'm just modifying my search method so that it can be used efficiently to search questions from multple places on the app